lib/internal/modules/esm/register.js

```diff
@@ -14,8 +16,9 @@ const { getHooksProxy } = require('internal/modules/esm/loader');
  */
 async function registerLoader(urlOrSpecifier) {
   const hooksProxy = getHooksProxy();
+  const parentURL = pathToFileURL(dirname(process.argv[1]) + sep).href
 
-  await hooksProxy.makeAsyncRequest('registerLoader', urlOrSpecifier)
+  await hooksProxy.makeAsyncRequest('registerLoader', urlOrSpecifier, parentURL)
```

Comment:
There’s no need for a `parentURL` argument. If someone wants to resolve a URL relative to a parent, they can do `registerLoader(new URL(pathToLoader, parent).href)`.